pom.xml

```diff
@@ -357,18 +342,32 @@
                         <configuration>
                             <dataFile>${jacoco-aggregate-destFile}</dataFile>
                             <rules>
-                                <rule >
+                                <rule>
                                     <element>CLASS</element>
                                     <limits>
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.97</minimum>
+                                            <minimum>0.5</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.92</minimum>
+                                            <minimum>0.59</minimum>
+                                        </limit>
+                                    </limits>
+                                </rule>
+                                <rule>
```

Comment:
Does this rule work at repo level?

Reply:
Terminated this but tldr: 
I was testing on my personal repository and accidentally pushed it where the idea was to set a different rule to the bundle level, or package level, or aka the repo level:
https://www.eclemma.org/jacoco/trunk/doc/check-mojo.html

But this actually didn't end up working, b/c when I set it to 99%, or 100%, or 200% it never failed, which is weird to me b/c it should fail if some of the classes are not fully covered. 

This example requires an overall instruction coverage of 80% and no class must be missed:

```
<rules>
  <rule>
    <element>BUNDLE</element>
    <limits>
      <limit>
        <counter>INSTRUCTION</counter>
        <value>COVEREDRATIO</value>
        <minimum>0.80</minimum>
      </limit>
      <limit>
        <counter>CLASS</counter>
        <value>MISSEDCOUNT</value>
        <maximum>0</maximum>
      </limit>
    </limits>
  </rule>
</rules>
```